server/controllers/updateEvent.js

```diff
@@ -1,13 +1,17 @@
 const { Event } = require('../database/models');
 
 exports.updateEvent = (req, res) => {
-  Event.update({ status: 1 }, { where: { id: req.params.id } })
-    .then((result) => {
-      if (result) {
-        res.json({ success: 'status updated' });
-      } else {
-        res.json({ sucess: 'status did not get updated!' });
-      }
-    })
-    .catch(() => res.status(500).json({ error: 'something wrong in query ' }));
+  if (req.userRole === 'admin') {
+    Event.update({ status: 1 }, { where: { id: req.params.id } })
+      .then((result) => {
+        if (result) {
+          res.json({ success: 'status updated' });
+        } else {
+          res.json({ sucess: 'status did not get updated!' });
```

Comment:
typo: `sucess` => `success`

I would have 2 messages in the JSON: `{ success: true/false, message: 'status (not) updated' }` and then in the front end you can just check success exists or not (rather than checking what it says)

Reply:
yup, done

Reply:
> typo: `sucess` => `success`
@m4v15 @shahenazmonia 
O.ops!
corrected